src/main/java/corgiaoc/byg/common/world/feature/overworld/iceburg/IceburgFeature.java

```diff
@@ -0,0 +1,64 @@
+package corgiaoc.byg.common.world.feature.overworld.iceburg;
+
+import com.mojang.serialization.Codec;
+import corgiaoc.byg.common.world.feature.config.SimpleBlockProviderConfig;
+import net.minecraft.block.Blocks;
+import net.minecraft.block.material.Material;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.ISeedReader;
+import net.minecraft.world.gen.ChunkGenerator;
+import net.minecraft.world.gen.Heightmap;
+import net.minecraft.world.gen.feature.Feature;
+import org.spongepowered.noise.module.source.Perlin;
```

Comment:
Remove this import as this library is difficult to export with the jar. Please use FastNoise instead(See floatings islands).